docs/use-cases/tokenization/real-world-assets.page.tsx

```diff
@@ -56,23 +57,29 @@ const companies = [
     link: "https://www.zoniqx.com/",
   },
   {
-    className: "archax",
-    alt: "Archax",
-    aspectRatio: 1.25,
-    link: "https://archax.com/",
+    className: "open-eden",
+    alt: "Open Eden",
+    aspectRatio: 5,
+    link: "https://openeden.com/tbill",
   },
   {
-    className: "meld",
-    alt: "Meld",
-    aspectRatio: 0.81,
-    link: "https://www.meld.gold/",
+    className: "ondo",
+    alt: "Ondo",
+    aspectRatio: 1.25,
+    link: "https://ondo.finance/",
   },
   {
     className: "ripple-logo",
     alt: "Ripple",
     aspectRatio: 3.89,
     link: "https://ripple.com/rwa-tokenization",
   },
+  {
+    className: "meld",
```

Comment:
What is the logic behind the order of the logos? They seemed to be in kind of a random order before and this changes them to be in a different random order?